index.html

```diff
@@ -8,6 +8,11 @@
   <link href="style.css" rel="stylesheet" type="text/css" />
   <script src="https://cdn.jsdelivr.net/npm/p5@1.4.1/lib/p5.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/p5@1.4.1/lib/addons/p5.sound.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/jquery@3.2.1/dist/jquery.min.js"></script>
+  <script src="
+https://cdn.jsdelivr.net/npm/jquery-csv@1.0.21/src/jquery.csv.min.js
```

Comment:
This was just the first parser I found, Jquery-csv that is. I added Jquery as well in case it needed anything from there, but I am not completely sure about that.